src/Seeder/BadasoContentPermissionsSeeder.php

```diff
@@ -19,8 +19,8 @@ public function run()
         foreach ($keys as $key) {
             Permission::firstOrCreate([
                 'key' => $key,
-                'table_name' => 'content',
                 'description' => 'Fill content',
+                'table_name' => null,
```

Comment:
why it's changed to be null, what's the different ? @miftahurrahmi

Reply:
because the badaso_contents table encountered difficulties when generating processes in the permission table. When set to null, the permission generation process succeeded

Reply:
what difficulties ? please explain more detail 

if table_name null, so how permission works ?